.github/workflows/go.yml

```diff
@@ -5,9 +5,8 @@ name: Go
 
 on:
   push:
-    branches-ignore: [ "main" ]
-  pull_request:
-    branches-ignore: [ "main" ]
+    tags: ['v[0-9]+.[0-9]+.[0-9]+']
+    branches: ['**']
 
 jobs:
   build_image_on_push:
```

Comment:
this should not reference a local reuseable workflow but later reference a workflow from https://github.com/dBildungsplattform/dbp-github-workflows/pull/1/files. We will talk about this in the Refinment today. After that PR is merged it can get referenced